androidtv/basetv/basetv.py

```diff
@@ -238,6 +238,20 @@ def _cmd_turn_on(self):
 
         return constants.CMD_TURN_ON_ANDROIDTV
 
+    def _cmd_audio_state(self):
```

Comment:
Could you please move this function above so that the `_cmd_*` functions are in alphabetical order.